swift/json_object_array.swift

```diff
@@ -0,0 +1,21 @@
+//Script writes an array of objects to stdout
+import Foundation
+
+guard CommandLine.arguments.count > 1 else {
+    print("Usage: swift script.swift <arg1> [<arg2> ...]")
+    exit(1)
+}
+
+let args = Array(CommandLine.arguments.dropFirst())
+
+let myArray = args.map { arg -> [String: Any] in
+    return [arg.uppercased(): arg.count]
+}
```

Comment:
```suggestion
let myArray = args.map { [$0.uppercased(): $0.count] }
```

We can
 - remove this unneeded type annotation
 - use `$0` to be more succinct